pkgs/development/python-modules/pytomlpp/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pybind11
+, pytestCheckHook
+, dateutil
+, doxygen
+, python
+, pelican
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "pytomlpp";
+  version = "0.3.5";
+  # pelican needs > 2.7
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "bobfang1992";
+    repo = pname;
+    rev = version;
+    fetchSubmodules = true;
+    sha256 = "1h06a2r0f5q4mml485113mn7a7585zmhqsk2p1apcybyydllcqda";
+  };
+
+  buildInputs = [ pybind11 ];
+
+  checkInputs = [
+    pytestCheckHook
+
+    dateutil
+    doxygen
+    python
+    pelican
+    matplotlib
+  ];
+
+  preCheck = ''
+    cd tests
+  '';
```

Comment:
Should work:

```suggestion
  pytestFlagsArray = [ "tests/" ];
```

Reply:
same as in the rtoml case, specifying `pytestFlagsArray` instead has the same effect as specifying neither